### DISCUSSION TITLE
Side effects of overlapping jsx components?

### DISCUSSION THREAD

**Discussion by ryuzakithe3rd** [pmndrs/react-spring#1755] [Support] [answered]
Hi new to react spring, and was trying to create a simple fade in/out transition on a background image, but there were side effects on non-animated components.

Here is a [sandbox](https://codesandbox.io/s/react-spring-animation-side-effects-7rr62?file=/src/App.js) replicating the problem.

Also tried searching and testing a lot of stuff and it seems the  CSS position property might be the cause, so tried a non background image and it works as expected.

Here is a [sandbox](https://codesandbox.io/s/no-side-effect-for-non-bg-image-63u1t?file=/src/App.js) showing the working non background image.

Any help on what is causing this behaviour and solutions will be greatly appreciated thanks :D

**Comment by joshuaellis** [comment, marked as answer]
This is just a CSS zindex-ing issue, see a resolution to the issue here – https://codesandbox.io/s/react-spring-animation-side-effects-forked-h0jyg?file=/src/App.js

**Reply by ryuzakithe3rd** [reply]
ah I see sorry for the misunderstanding and thanks you so much for the help :)
